src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/LegacyCaseRepository.java

```diff
@@ -42,4 +42,12 @@ Integer linkDefendant(
         @Bind("defendantId") String defendantId
     );
 
+    @SqlUpdate("UPDATE claim SET "
+        + " county_court_judgment = :countyCourtJudgmentData::JSONB,"
```

Comment:
put back in claimRepository

Reply:
Moved there in legacy, as when we will switch off claim-store, the legacy code to remove will be in same place and will be easy to turn off on some condition as well.